### PR TITLE
Negative padding for functional_pil #2381

### PR DESCRIPTION
**Pull request by jamt9000**
Fix as discussed in #2381 (tests to follow)

### CODE REVIEW

File: test/test_transforms.py [hunk 1]
```diff
@@ -398,6 +408,25 @@ def test_pad_with_mode_F_images(self):
         padded_img = transform(img)
         self.assertSequenceEqual(padded_img.size, [edge_size + 2 * pad for edge_size in img.size])
 
+    def test_pad_negative(self):
```
Comment (vfdev-5) [inline test/test_transforms.py:411]: Can you put and adapt this test into https://github.com/pytorch/vision/blob/b5664ea82405c37b1f394bf4f70d6ecef3a35b11/test/test_transforms_tensor.py#L123

**Review by vfdev-5 — COMMENTED** [review]
@jamt9000 thanks for the PR ! Looks good. 
I left  a comment to change the test.

File: test/test_transforms.py (hunk 1 above)
Comment (vfdev-5) [reply test/test_transforms.py]: @jamt9000 it is not the same `test_pad` :) It is one inside `test_transforms_tensor.py`. Thanks !

Comment (jamt9000) [reply test/test_transforms.py]: Oops, changed the right one now

**Review by vfdev-5 — APPROVED** [review]
LGTM! Thanks @jamt9000

**Pull request merged**